.github/workflows/docs.yaml

```
@@ -0,0 +1,40 @@
+name: Quarto documents
+
+on:
+  pull_request:
```

Should probably run only on PRs to main, no?

```suggestion
  pull_request:
     branches: [main]

---

What about production/tagging whatever we end up calling the future branches? We can leave it to the main only, but I just wanted to comment on that so we have it on the back of our minds!

---

Yes, I think if in the future we have other key persistent branches those should be added here.